Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -18,17 +18,29 @@ import Foundation
 import CircuitBreaker
 
 /// Object containing everything needed to build HTTP requests and execute them
-public class RestRequest {
-
+public class RestRequest: NSObject  {
+    
+    // Check if there exists a self-signed certificate and whether it's a secure connection
+    private var isSecure: Bool = false
+    private var isSelfSigned: Bool = false
```

Comment:
@ShihabMehboob Can we make `isSelfSigned ` a constant instead? Looking at the code, it seems we should be able to do that: `private let isSelfSigned: Bool`... and then assign the corresponding value in the initializer.